teamA/lib/Api/moodle_api_singleton.dart

```diff
@@ -118,7 +118,7 @@ class MoodleApiSingleton {
     return courses;
   }
 
-*/
+/*
```

Comment:
Is this code needed but not currently used? If not can it be removed rather than commented please

Reply:
Removed the commented code, its good now.